c2corg_ui/templates/helpers/list.html

```diff
@@ -3,6 +3,7 @@
     <option translate value="${request.route_url('routes_index')}" ng-selected="{{mainCtrl.isPath('routes')}}">Routes</option>
     <option translate value="${request.route_url('waypoints_index')}" ng-selected="{{mainCtrl.isPath('waypoints')}}">Waypoints</option>
     <option translate value="${request.route_url('outings_index')}" ng-selected="{{mainCtrl.isPath('outings')}}">Outings</option>
+    <option translate value="${request.route_url('articles_index')}" ng-selected="{{mainCtrl.isPath('articles')}}">Articles</option>
```

Comment:
I don't think we'll add articles to this list: they have their own entry in the sidemenu. https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/sidemenu.html#L37-L40